index.js

```diff
@@ -615,8 +615,15 @@ async function getRepos(page = 1) {
  * @returns The manifest object
  */
 async function getRepoManifest(user, repo, branch) {
-    const url = `https://raw.githubusercontent.com/${user}/${repo}/${branch}/manifest.json`;
+    if (window.sessionStorage.getItem(user + "-" + repo)) {
+        return JSON.parse(window.sessionStorage.getItem(user + "-" + repo));
+    }
 
+    const url = `https://raw.githubusercontent.com/${user}/${repo}/${branch}/manifest.json`;
+    const unloadable = getUnloadableManifests();
+    if ( unloadable.includes(url)) {
+        return null;
+    }
     return await fetch(url).then(res => res.json()).catch(() => null);
```

Comment:
This should probably cache the value to sessionStorage as well, after it's go it, to reduce calls.